brion/morphologyPlugin.h

```diff
@@ -144,6 +141,9 @@ class MorphologyPlugin : public boost::noncopyable
     /** @copydoc brion::Morphology::flush */
     virtual void flush() = 0;
     //@}
+
+protected:
+    MorphologyInitData _data;
```

Comment:
This is a lot of code changes for not so much gain, true the code is shorter, but I guess you spent more time changing the code than having added the data fields to the upcoming morphology plugin.

I'd prefer if you store just the family and version instead of the full data. Sure, the URI is small, but if it's not needed, why keeping it?